kaitaistruct.py

```diff
@@ -1,18 +1,19 @@
 import itertools
 import sys
 import struct
+import zlib
```

Comment:
No, it shouldn't be here. Instead https://github.com/kaitai-io/kaitai_compress/pull/2 (and maybe its successor https://github.com/KOLANICH/transformerz.py/tree/master/transformerz/kaitai ) should be used ... in a generic way. In fact it really requires the interfaces proposal being implemented. So, no compression in serialization for now.

Reply:
Please, for the love of god add serialization option to either of those, then and ship them with this runtime module. It feels very clumsy to have to write the code for those from scratch each time or install different module for the same single purpose when the compression/decompression is clearly supported by the ksy format natively.

Reply:
I don't understand what you mean. My modules define interface of 3 functions for compressor/decompressors and implement them. And they are actually used in the wild. I use my module to compress a database.

Reply:
I mean wouldn't it be better if I could just run `pip install kaitaistruct` and use the decoder/encoder right away?

Reply:
No. This stuff is optional.

Reply:
When targetting Python, the compiler currently converts `process: zlib` into a `zlib.decompress` call directly in the generated code. As far as I can tell, the same approach would work with serialization, so there should be no need for extra functions in the runtime for this.

@KOLANICH This really isn't the place to propose major changes to how the standard KS `process` functions should be handled. At the moment, the compiler has native support for `process: zlib` when parsing, so it should be the same for serialization. If you want to change that, please discuss that in another issue.

Reply:
But `zlib` mentioned in the `process` options on the main kaitai.io site, how can it be optional?

Reply:
What I meant when adding that is that compiller should probably reference runtime, not write code directly each time like it does now. But this isn't really place to discuss this, after some deliberation. 

If compiler don't reference it why do we need runtime in the first place, because compiler can just write `struct`s with all needed struct-format and service functions directly into resulting file which will be arguably more convenient to use as format classes would be self-contained then.